test/extensions/filters/network/http_connection_manager/config_test.cc

```diff
@@ -1708,8 +1707,30 @@ TEST_F(HttpConnectionManagerConfigTest, DefaultRequestIDExtension) {
                                      date_provider_, route_config_provider_manager_,
                                      scoped_routes_config_provider_manager_, http_tracer_manager_,
                                      filter_config_provider_manager_);
-  auto request_id_extension =
-      dynamic_cast<Http::UUIDRequestIDExtension*>(config.requestIDExtension().get());
+  auto request_id_extension = dynamic_cast<Extensions::RequestId::UUIDRequestIDExtension*>(
+      config.requestIDExtension().get());
+  ASSERT_NE(nullptr, request_id_extension);
```

Comment:
nit: Here, and in `DefaultRequestIDExtensionWithParams` test: would be nice to be able to verify config particulars (even though it's just one parameter atm), not just that the extension has been instantiated.

Reply:
Fixed